### DISCUSSION TITLE
let certain columns be header filtered, and others have no filtering

### DISCUSSION THREAD

**Discussion by zackdillabough** [komarovalexander/ka-table#223] [Q&A] [answered]
Hello,

Is it currently possible to specify certain columns (grouped columns, in particular) to have filteringMode: FilteringMode.None, and then let all others have filteringMode: FilteringMode.HeaderFilter?

Thank you

**Comment by komarovalexander** [comment, marked as answer]
Hi @zackdillabough 
there is no option to set filteringMode for certain column,
But I just released 7.4.1 version which provides the ability to customize headerFilterButton
https://github.com/komarovalexander/ka-table/releases/tag/7.4.1

and I've updated the demo https://komarovalexander.github.io/ka-table/#/header-filter so you can use this solution for your task

**Reply by zackdillabough** [reply]
Awesome, thank you very much Alexander!
